extension/tests/integration_tests.rs

```diff
@@ -52,6 +52,60 @@ async fn test_scheduled_job() {
     assert_eq!(result.rows_affected(), 3);
 }
 
+#[ignore]
+#[tokio::test]
+async fn test_drop_table_triggers_job_deletion() {
+    let conn = common::init_database().await;
+    let mut rng = rand::thread_rng();
+    let test_num = rng.gen_range(1..100000);
+    let test_table_name = format!("drop_test_table_{}", test_num);
+    let job_name = format!("job_{}", test_num);
+
+    common::init_test_table(&test_table_name, &conn).await;
+
+    let insert_job_query = format!(
+        "INSERT INTO vectorize.job (name, index_dist_type, transformer, search_alg, params, last_completion)
+        VALUES ('{job_name}', 'pgv_hsnw_cosine', 'sentence-transformers/all-MiniLM-L6-v2', 'search_algorithm', 
+                jsonb_build_object('table', '{test_table_name}', 'schema', 'public'), NOW());"
+    );
```

Comment:
This should be a `select vectorize.table(...)` instead, so that the test stays current with the APIs around that table.

Reply:
@varshith257 - i think replace this direct insert with `vectorize.table()`. It should help the test move along as it is currently failing because the search_alg column no longer exists (it was removed from the project).